### PR TITLE
SSC1-8 login form component

### PR DESCRIPTION
**Pull request by akhil-maram**
Create LoginForm component with passing tests

https://akhilmaram.atlassian.net/browse/SSC1-38
https://akhilmaram.atlassian.net/browse/SSC1-8
Screenshot 2024-07-19 at 12 33 24

### CODE REVIEW

File: frontend/src/Components/LoginForm.jsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import "../resources/LoginForm.css";
+
+const LoginForm = () => {
+  return (
+    <div className='form-container'>
+        <h1>Welcome to SkySocial</h1>
```
Comment (Dhyey618) [inline frontend/src/Components/LoginForm.jsx]: add a space between the sky and social

File: frontend/src/Components/LoginForm.jsx
Comment (dilpreetshoker04) [inline frontend/src/Components/LoginForm.jsx:1]: Classnames such as "input container" are too generic

**Review by Dhyey618 — COMMENTED** [review]
add a space between sky and social at the welcome to SkySocial

File: frontend/src/Components/LoginForm.jsx
Comment (dilpreetshoker04) [reply frontend/src/Components/LoginForm.jsx:1]: same applies to "input-field" and "form-control". Similar names may be used in Create event form and other forms throughout the project and may cause issues in the future.

File: frontend/src/resources/LoginForm.css [hunk 2]
```diff
@@ -0,0 +1,21 @@
+.input-field {
+    margin-bottom: 2%;
+}
+
+.input-container {
+    align-items: center;
```
Comment (dilpreetshoker04) [inline frontend/src/resources/LoginForm.css:6]: Is this line needed

File: frontend/src/resources/LoginForm.css [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    width: 40%;
+    margin-left: 30%;
+}
+
+.form-container {
+    align-content: center;
+}
+
+.create-account-container {
+    margin: 1em;
```
Comment (dilpreetshoker04) [inline frontend/src/resources/LoginForm.css]: Do we know exactly what 1em is. I don't but if we have control over this then it's okay. I normally like working in pixels and and percentages where necessary but yeah that's personal preference.

**Review by dilpreetshoker04 — CHANGES_REQUESTED** [review]

File: frontend/src/resources/LoginForm.css (hunk 3 above)
Comment (akhil-maram) [reply frontend/src/resources/LoginForm.css]: 1em has been changed to 30px.

File: frontend/src/resources/LoginForm.css (hunk 2 above)
Comment (akhil-maram) [reply frontend/src/resources/LoginForm.css:6]: This container is needed for positioning the input fields in the center.

File: frontend/src/Components/LoginForm.jsx
Comment (akhil-maram) [reply frontend/src/Components/LoginForm.jsx:1]: The "form-control" was provided by CSS and used in our code. As per the other classNames, these have been renamed to fit the appropriate conventions.

File: frontend/src/Components/LoginForm.jsx (hunk 1 above)
Comment (akhil-maram) [reply frontend/src/Components/LoginForm.jsx]: Space has been added

**Review by dilpreetshoker04 — APPROVED** [review]

**Review by Dhyey618 — APPROVED** [review]
GTG

**Pull request merged**